app/models/manageiq/providers/openstack/inventory/parser/cloud_manager.rb

```diff
@@ -146,6 +147,43 @@ def host_aggregates
     end
   end
 
+  def placement_group_by_vm_id
+    @placement_group_by_vm_id ||= collector.placement_groups.each_with_object({}) { |sg, result| sg.members.each { |vm_id| result[vm_id] = sg } }
+  end
+
+  def placement_groups
+    collector.placement_groups.each do |spgrp|
+      pgrp         = persister.placement_groups.find_or_build(spgrp.name)
+      pgrp.name    = spgrp.name
+      pgrp.ems_ref = spgrp.id
+      pgrp.policy  = spgrp.policies[0]
```

Comment:
Should placement_group policies be its own table if there can be multiple?

Reply:
right now it is just one either affinity or anti-affinity.

Reply:
Okay can you comment here that there can only be one otherwise arbitrarily taking the first looks suspicious